src/bin/bindgen.rs

```diff
@@ -214,12 +215,14 @@ pub fn main() {
     let mut bind_args: Vec<_> = env::args().collect();
     let bin = bind_args.remove(0);
 
-    match bindgen::get_include_dir() {
-        Some(path) => {
-            bind_args.push("-I".to_owned());
-            bind_args.push(path);
+    if let Some(clang) = clang_sys::support::Clang::find(None) {
+        // TODO: distinguish C and C++ paths? C++'s should be enough, I guess.
+        for path in clang.cpp_search_paths.into_iter() {
+            if let Ok(path) = path.into_os_string().into_string() {
+                bind_args.push("-idirafter".to_owned());
```

Comment:
Every script we wrote used `-isystem` write? I think we should keep doing this.

What is upstream in that discussion btw? And why do they have to still massage the search path? Did anyone look into why libclang doesn't behave the same way as clang on that front?